content/Contributing and Debugging/_index.md

```diff
@@ -3,20 +3,20 @@ weight: 13
 title: Contributing and Debugging
 ---
 
-# Contributing guidelines
+## Contributing guidelines
```

Comment:
This could simply be removed. From what I can tell it's the only h1 in the entire file. With this there's no need to change the other headings.